internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -64,6 +105,24 @@ func shouldCPULoadBalancingBeDisabled(annotations fields.Set) bool {
 	return value == "true"
 }
 
+func shouldCPUQuotaBeDisabled(annotations fields.Set) bool {
+	value, ok := annotations[annotationCPUQuota]
+	if !ok {
+		return false
+	}
+
+	return value == "true"
+}
+
+func shouldIRQLoadBalancingBeDisabled(annotations fields.Set) bool {
+	value, ok := annotations[annotationIRQLoadBalancing]
+	if !ok {
+		return false
+	}
+
+	return value == "true"
```

Comment:
ditto

Reply:
I see that it's just a copy-paste, so not really your code.

What you can do is add a first commit which simplifies the existing code (by removing the `ok` check), then add your own commit on top of it.

Reply:
Addressed some of these in https://github.com/cri-o/cri-o/pull/4026, please feel free to cherry-pick (or wait for that one to be merged).

Reply:
Thanks for the #4026 cleanup. I will add some tests while waiting that one merged